arch/arm/src/stm32f7/stm32f74xx75xx_rcc.c

```diff
@@ -1005,3 +1006,44 @@ static inline void rcc_enableperipherals(void)
 /****************************************************************************
  * Public Functions
  ****************************************************************************/
+
+/****************************************************************************
+ * Name: stm32_rcc_dump_regs
+ *
+ * Description:
+ *   Dump the contents of all rcc block registers
+ *
+ * Input Parameters:
+ *   msg - Message to print before the register data
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_DEBUG_INFO
+void stm32_rcc_dump_regs(const char *msg)
```

Comment:
@ChaochaoCui nobody call stm32_rcc_dump_regs, why not directly remove it?

Reply:
for debug. I will remove it.